app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+<!-- -->>read3/1|3
+<!-- -->332163 332163 332163 2 1 332163 2 332163 332163 332163 332163 332163 1 332163 332163 332163 332163 332163 332163 332163 332163 332163
+<!-- -->>read3/1|1R
+<!-- -->
+<!-- -->>read3/1|2R
+<!-- -->
+<!-- -->>read3/1|3R
+<!-- -->
+<!-- -->>read3/2|1
+<!-- -->
+<!-- -->>read3/2|2
+<!-- -->
+<!-- -->>read3/2|3
+<!-- -->
+<!-- -->>read3/2|1R
+<!-- -->332163 2 2 332163 332163 332163 332163 332163 332163 332163 1 332163 332163 332163 2 2 1 332163 332163
+<!-- -->>read3/2|2R
+<!-- -->5855 1 1 2 519963 5206
+<!-- -->>read3/2|3R
+<!-- -->
+        </pre>
         <p>
-          This allows us to determine the most abundant species-level identifications in the sample <i>(timing: 0 secs)</i>.
-    </p>
+          Note: the real output would contain newlines between taxa,
+          which have been replaced here by spaces for readability.
+        </p>
         <pre>
-<b>$</b> join -t, -1 1 -2 1 distribution.csv taxons.csv | sort -t, -k2,2nr | grep species | head -n30</pre>
+<!--   --><b>$</b> mv selected-seeds.fa found-kmers.fa # optional step, same format
+        </pre>
+        <p>
+          This algorithm requires spatial information which is provided
+          by supplying the <code>-o</code> flag to the previous command.
+          The <code>taxons.tsv</code> is a file describing the NCBI taxonomic tree.
+        </p>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Aggregation</h2></div>
+      <div class="card-supporting-text">
         <p>
-          The 30 most dominant species are:
-    </p>
+          At this step of the pipeline, we have a list of taxon identifiers per protein read,
+          one protein read per reading frame and two paired-ends per metagenomics read.
+          To get a single taxon identification per read, we need to aggregate this list.
+          UMGAP offers 3 alternative aggregation strategies with varying speciality.
+
+          <ul>
+            <li>
+              Unipept's lowest common ancestor strategy (LCA*),
+              where the consensus taxon is, as before, the taxon of most specific rank
+              which is either descendant or ancestor of every given taxon;
+            </li>
+            <li>
+              Kraken's maximum root-to-leaf path (MRTL),
+              where the consensus taxon is the taxon in the given list
+              which has the most ancestors in the given list;
+            </li>
+            <li>
+              A newly developed hybrid strategy,
+              which combines LCA* and MRTL based on a given scaling factor (default 0.25).
+              The hybrid method aims to ignore some outliers without compromising generality.
+              This strategy is the default and will therefore be used below.
+            </li>
+          </ul>
+        </p>
+        <p>
+          Before this aggregation, we remove the markers from the headers with <code>sed</code>
```

Comment:
Before this aggregation -> Before aggregating the results per read

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16559) by @bmesuere on Mon Aug 20 2018 at 14:06._